### ISSUE TITLE
Request for Enhancement: Notification Pop-up on Tutorial Publish/Unpublish

### ISSUE THREAD

**Issue by skbhagat0502** [scorelab/Codelabz#886]
**Description:**

I would like to bring your attention a potential enhancement in the Codelabz website regarding the publishing and unpublishing of tutorials. User do not gets any alerts that the tutorial is published or not.

**Issue:**

Currently, when a tutorial is published or unpublished, there is no immediate feedback provided to the user. This lack of notification might lead to a less-than-optimal user experience as users may not be aware of the successful publication of their tutorial.

How to raise the issue:

- Create a new codelabz.
- Write the steps.
- Publish the tutorial.
- Unpublish the tutorial.

**Suggested Enhancement:**

I propose implementing a notification pop-up/modal that appears upon successful tutorial publication. This pop-up would serve to inform the user that the tutorial has been published successfully and provide options to either view the published tutorial or create another one.

**Technical Implementation:**

Considering the variety of approaches available for creating pop-ups in a React-based application, I would like to discuss the preferred method for implementation. Options include using a React Alert library, a custom modal component, or leveraging Material-UI's modal functionality. Each approach has its merits, and I am open to your guidance on which method aligns best with the existing codebase and design principles.

**Comment by skbhagat0502** [comment]
@shivareddy6 @Sarfraz-droid Can I work on this. If yes please assign me this also tell which type of alert should be implemented.

**Comment by shivareddy6** [comment]
Try to build something very modular which be used everywhere in the application (from small errors to acknowledgement messages). Research upon few versions of toast notifications and suggest your plan of action here.

**Comment by skbhagat0502** [comment]
Hi @shivareddy6 , after reviewing the various versions of react-toastify, I noticed that the newest version is compatible with React versions greater than 16.8. Given that the React version on the Codelabz website surpasses 16.8, I believe opting for the latest version is the optimal choice. This version offers versatility, allowing us to handle error messages, success messages,  warnings or info messages with ease. Another advantage is its seamless integration across the entire website with minimal code required for each component. I've attached a video showcasing react-toastify for your reference. Kindly tell me your point of view if I can start working on it with this version.

https://github.com/scorelab/Codelabz/assets/109683163/cb34aacd-5959-488c-8fa7-60234c3673f3

Reference website link : [https://fkhadra.github.io/react-toastify/introduction](https://fkhadra.github.io/react-toastify/introduction)
